src/main/java/org/opengis/cite/ogcapiedr10/collections/CubeCollections.java

```diff
@@ -48,17 +48,17 @@ public class CubeCollections extends AbstractFeatures {
 	CollectionsTime ct= new CollectionsTime();
 	/**
 	 * <pre>
-	 * Abstract Test 70: Validate that the coords query parameters are constructed correctly. (cube)
+	 * Abstract Test 70: Validate that the bbox query parameters are constructed correctly. (cube)
```

Comment:
The standard (https://docs.ogc.org/is/19-086r4/19-086r4.html) still uses the old (wrong) sentence. I prefer it to leave it like that and to add a short comment explaining that the abstract test of the specification is wrong so that the test is implemented differently.